tests/unit/bokeh/protocol/messages/test_patch_doc.py

```diff
@@ -58,6 +58,7 @@ def _sample_doc(self):
         another = AnotherModelInTestPatchDoc()
         doc.add_root(SomeModelInTestPatchDoc(child=another))
         doc.add_root(SomeModelInTestPatchDoc())
+        doc.to_json() # clear new model queue
```

Comment:
It should be possible to have a unit test of `flush_synced` that directly exercises the new `is_still_new` parameter, I think?